ui/tests/e2e-1000-genomes/test.js

```diff
@@ -7,22 +7,22 @@ jasmine.getEnv().addReporter({
   }
 });
 
-describe("End-to-end", () => {
+describe("End-to-end 1000 genomes", () => {
   beforeAll(async () => {
     // It can take a while for servers to start up
     jest.setTimeout(JEST_TIMEOUT_MS);
     await waitForElasticsearchIndex();
 
     // Hide snackbar because it prevents clicking on some facet bars
-    await page.goto("http://localhost:4400", {timeout: 120000});
+    await page.goto("http://localhost:4400", { timeout: 120000 });
```

Comment:
The timeouts on 17 and 25 aren't needed right? We were just trying to fix the circleci problem.